automation/taskcluster/decision_task.py

```diff
@@ -320,11 +303,23 @@ def linux_build_task(name):
             "application-services-rustup": "/root/.rustup",
             "application-services-android-ndk-toolchain": "/root/.android-ndk-r15c-toolchain",
         })
-        .with_index_and_artifacts_expire_in(build_artifacts_expire_in)
+        .with_index_and_artifacts_expire_in("1 month")
```

Comment:
Regression: I don't see indexes being taken care of in the taskgraph patch. Are developers fine to not have indexes at the moment?

Reply:
Routes were only used for `libs` (now covered by toolchain builds) and docker images (now covered by taskgraph docker image building).
See [the `project.application-services.application-services` index](https://tools.taskcluster.net/index/project.application-services.application-services)